docker/base.dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM ubuntu:bionic
```

Comment:
```suggestion
FROM ubuntu:jammy
```

`bionic` is Ubuntu 18.04, which reached the end of standard support in June. `jammy` is 22.04

updating this might negate the need to make symbolic links for `python` and `pip`, and `pip3` might be the default for `pip`?

Reply:
@bjester After updating to ubuntu:jammy  I am getting this error. 
<img width="740" alt="Screenshot 2023-10-16 at 9 17 56 PM" src="https://github.com/learningequality/kolibri/assets/117556787/3e7c8b4e-5151-4758-b5e9-e6322a2c19b1">
After a bit of googling, it seems that for Python 3.9, the 'HTMLParser' object has no attribute 'unescape'. So should I try downgrading Python ?

Reply:
For the development Dockerfile, we should remove the installation of the `build.txt` requirements file - this is Python 2.7 specific, so causes issues on newer versions of Python.

Reply:
so we should remove the "pip3 install -r requirements/build.txt" from the base.dockerfile ?

Reply:
Yes remove the install of the `requirements/build.txt` file. It will only cause more issues

Reply:
okey, thanks.

Reply:
I have updated the pull request with the changes. But still it requires symbolic links .
$ make docker-devserver                     (without symbolic links)
<img width="583" alt="Screenshot 2023-10-17 at 12 35 45 AM" src="https://github.com/learningequality/kolibri/assets/117556787/5dcf45c6-64e8-4669-942e-2f2944603d57">
after adding symbolic links it works fine.
<img width="516" alt="Screenshot 2023-10-17 at 1 09 04 AM" src="https://github.com/learningequality/kolibri/assets/117556787/990be902-a986-4914-b42c-a0dd093f2e42">